docs/source/events/examples/donation.yaml

```diff
@@ -0,0 +1,10 @@
+id: 123
+type: donation
+version: 2.1.0
+date: '2021-05-14T08:07:37'
+amount: 99.99 EUR
```

Comment:
Should currency as a standalone field be added here?

Reply:
I would rather migrate to combined fields in the payment_status_change events as well.